dataland-frontend/src/components/resources/frameworkDataSearch/FrameworkDataSearchResults.vue

```diff
@@ -13,6 +13,7 @@
       @row-click="goToData"
       class="table-cursor"
       id="search-result-framework-data"
+      data-test="search-result-framework-data"
```

Comment:
This can be reverted to main too.
I think you implemented this to click on a search result on the search page right?
But you don't need that in my opinion.
You can directly visit the view-page in your editButtonTest-test instead of clicking on the search result here.

Hit me up if you need more input on this.

Reply:
Done.

Reply:
👍